crates/nargo/src/cli/mod.rs

```diff
@@ -82,6 +82,7 @@ pub fn start_cli() {
             App::new("prove")
                 .about("Create proof for this program")
                 .arg(Arg::with_name("proof_name").help("The name of the proof"))
+                .arg(Arg::with_name("checked").help("Verify proof after proving"))
```

Comment:
There seems to be precedence for the `--checked` name, but wouldn't `--verify` or similar be clearer?